shardingsphere-distribution/shardingsphere-proxy-native-distribution/Dockerfile

```diff
@@ -19,7 +19,7 @@ FROM busybox AS prepare
 
 RUN mkdir -p /conf/
 
-FROM scratch
+FROM debian:bullseye-slim
```

Comment:
- According to https://www.graalvm.org/22.2/reference-manual/native-image/guides/containerise-native-executable-and-run-in-docker-container/ and https://github.com/graalvm/graalvm-demos/blob/master/spring-native-image/Dockerfiles/Dockerfile Behaviour , fix for `Dockerfile`, as default configuration doesn't have the ability to only use `scratch` as the base docker image. 
- I just chose `debian:bullseye-slim` which does not require frequent updates among `debian:bullseye-slim` and `oraclelinux:9-slim`.

Reply:
- Should we sync the GraalVM example, using `oraclelinux:9-slim`?